src/Microsoft.Health.Dicom.Core/Features/Workitem/AddWorkitemRequestHandler.cs

```diff
@@ -47,12 +44,8 @@ public class AddWorkitemRequestHandler : BaseHandler, IRequestHandler<AddWorkite
 
         request.Validate();
 
-        var workitems = await _workitemSerializer
```

Comment:
Do we still need the `IWorkitemSerializer` interface?

Reply:
Done

Reply:
I still don't see this removal